src/utilities.js

```diff
@@ -0,0 +1,20 @@
+import R from 'ramda';
+import { NpmConfig, httpsGetPromise } from './helpers';
+
+const removeCaret = R.replace(/\^/, '');
+const addVersionProp = (v, k, o) => o[k] = { 'version': removeCaret(v) };
+const isNotPrivate = R.compose(R.not, R.prop('private'));
+const deeplyMerge = (obj1, obj2) => R.mapObjIndexed((v, k, o) => R.merge(obj1[k], obj2[k]), obj1);
+
+export default {
```

Comment:
Let's go ahead and add the remainder of the exported functions to this object for consistency's sake.